Update msi-tutorial-linux-vm-access-storage.md

AngshumanNayakMSFT opened the pull request:
Curl doesn’t translate the query parameter “?api-version=2017-11-09” in the URL into the header. Adding -H “x-ms-version: 2017-11-09” to curl command makes it work.

azurecla commented:
Hi __@bjtucker__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

bjtucker commented:
Nothin but love for you, robot, but with all due respect, I don't think you're right that a cla would be required here. I transcribed what was in the images. That's not my own code, it's exactly  what was in the pictures. :)

I see lots of similar edits tagged with no CLA required. Did the robot just get scared when it saw some curly braces, or do we actually need a CLA here?

Jim-Parker commented:
@bjtucker Ben, first, thanks for the contribution. I can see your point, but in this case the requirement is based on scope of change, not the content of the change. We don’t have discretion on this. You'll need to sign the CLA for us to proceed. Thanks.

bjtucker commented:
Closing this out, since I'm going on vacation and I'm not going to be able to address this ahead of that.
I can't possibly have copyright over a literal transcription of these images--i've done zero creative work here--so I don't think I'm doing harm by just trashing this pull request. If it's a good idea, someone else can just submit the same.

I'll get the CLA signed before I try again. Thanks!

AngshumanNayakMSFT committed:
Update msi-tutorial-linux-vm-access-storage.md

Curl doesn’t translate the query parameter “?api-version=2017-11-09” in the URL into the header. Adding -H “x-ms-version: 2017-11-09” to curl command makes it work.
